### PR TITLE
Fix All Recent CVEs

### PR DESCRIPTION
**Pull request by sumansreenivasan**
### JIRA link ###

https://tools.hmcts.net/jira/browse/RDM-10583

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -120,7 +120,8 @@
     "eslint": "^4.18.2",
     "bl": "^4.0.3",
     "dot-prop": "^4.2.1",
-    "node-fetch": "^2.6.1"
+    "node-fetch": "^2.6.1",
+    "ini": "1.3.7"
```
Comment (MSancaktutar) [inline package.json:124]: yarn lock should change

Comment (sumansreenivasan) [reply package.json:124]: Added updated yarn.lock

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Comment by MSancaktutar** [comment]
Merged with confirmation with Ola.

**Pull request merged**